tests/helpers/drupal-commands.js

```diff
@@ -24,6 +24,17 @@ module.exports = {
     const result = drush(`test:node-get-id "${node_title}"`);
     return await page.goto('/node/' + result.toString() + '/layout');
   },
+  
+  /**
+   * Finds node ID via drush and visits node layout preview page.
+   * @param  {Array<{page: Page, node_title: String}>} array Page object and
+   *   node title
+   * @return {Response} The response.
+   */
+  visitNodeLayoutPreviewPage: async ([page, node_title]) => {
+    const result = drush(`test:node-get-id "${node_title}"`);
+    return await page.goto('/node/' + result.toString() + '/layout-preview?auth=1');
```

Comment:
this is confusing me, is this going to the frontend or the backend? how does this work reliably?

Reply:
it doesn't really matter because it will be redirected if there is no /api in the path.

Reply:
I see. I think this is probably good to test also, so let's simply keep it like this. would be worth a comment though, because that's not obvious